kalman_watch.py

```diff
@@ -46,7 +46,10 @@ def get_opt():
 opt = get_opt()
 
 stop = DateTime(opt.stop)
-start = DateTime(opt.start or stop - 3 * 365)
+
+# Default start is beginning of calendar year 3 years ago
+start = DateTime(opt.start or DateTime(int(DateTime().frac_year - 3), format='frac_year').date)
```

Comment:
This doesn't satisfy the "have at least 3.5 years of data" suggestion, but it does have it roll over to give us at least 3 years of data with a fixed calendar year start.